package.json

```diff
@@ -107,7 +107,7 @@
     "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#77cc9d42c474a693755941b10ee4c2d6f50e5346",
     "react-native-plaid-link-sdk": "^7.2.0",
     "react-native-quick-sqlite": "^5.0.3",
-    "react-native-reanimated": "3.0.0-rc.6",
+    "react-native-reanimated": "2.14.4",
```

Comment:
I think we want to use reanimated `3.0.0-rc.10` instead, based on the conversation in Slack [here](https://expensify.slack.com/archives/C03TQ48KC/p1675812945076659).